apps/gateway/src/main/resources/application.yaml

```
@@ -1,11 +1,18 @@
     port: 8084
 eureka:
     client:
         service-url:
             defaultZone: http://localhost:8761/eureka/
         register-with-eureka: true
-        fetch-registry: true
\ No newline at end of file
+        fetch-registry: true
+logging:
+    level:
+      org:
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)